src/main/java/org/example/bookstoremate/controller/BookController.java

```diff
@@ -0,0 +1,35 @@
+package org.example.bookstoremate.controller;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.example.bookstoremate.dto.BookDto;
+import org.example.bookstoremate.dto.CreateBookRequestDto;
+import org.example.bookstoremate.service.BookService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/books")
```

Comment:
```suggestion
@RequestMapping("/books")
```

add `server.servlet.context-path=/api` to application.properties